docs/content/integration/trusted-header-sso/seerr/index.md

```
@@ -0,0 +1,98 @@
+* This feature will only work if a list of trusted proxies is provided.
+
+Some of the values presented in this guide can automatically be replaced with documentation variables.
+
+{{< sitevar-preferencer >}}
+
+## Configuration
+
+To configure [Seerr] to trust the `Remote-User` and `Remote-Email` header do the following:
+
+### With GUI
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)